README.md

```diff
@@ -49,8 +49,7 @@ https://github.com/jasonmerino/react-native-simple-store
 # realm-react-native/
 Realm React Native enables you to efficiently write your app’s model layer in a safe, persisted and fast way. Here’s what it looks like: See Realm React Native and Docs For Realm
 read more
-https://realm.io/news/introducing-realm-react-native/
-https://realm.io/news/introducing-realm-react-native/
```

Comment:
These URLs redirect to https://blog.realm.io/blog/introducing-realm-react-native/.
The https://blog.realm.io/ is downs.